doc/conf.py

```diff
@@ -0,0 +1,82 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+import os
+import sys
+import inspect
+
+__location__ = os.path.join(os.getcwd(), os.path.dirname(
+    inspect.getfile(inspect.currentframe())))
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+sys.path.insert(0, os.path.join(__location__, '../src'))
+module_dir = os.path.join(__location__, "../src/osm_flex")
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'OSM-flex'
+copyright = '2023, authors'
+logo_only = True
+html_logo = 'logo_osm-flex.png'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.duration",
+    "sphinx.ext.doctest",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.viewcode",
+    "autoapi.extension",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.intersphinx",
+    "nbsphinx"   
```

Comment:
I recommend using `myst-nb` instead, it can be installed via pip. `nbsphinx` requires conda due to its dependencies. See the latest CLIMADA doc configuration.

Reply:
Does `myst-nb` work the same? Or what else would then have to be amended?

Reply:
It seems to work, so it might be better not to change it. Maybe `nbsphinx` is "pre-installed" in readthedocs environments? It's all a bit weird over there. Great to see that it works, though 😅

Reply:
oki, then lets leave it at that for now